.github/github-runner-on-aws-ec2-spot/.gitignore

```diff
@@ -0,0 +1,8 @@
+*.log
+*.terraform.lock.hcl
+terraform.tfstate
```

Comment:
You are working in a team, use `Backend Type: s3`,  don't store tfstate locally.

Reply:
to be disscussed with voice